tests/e2e/csi_snapshot_basic.go

```diff
@@ -190,12 +190,23 @@ var _ = ginkgo.Describe("[block-vanilla-snapshot] Volume Snapshot Basic Test", f
 				err := snapc.SnapshotV1().VolumeSnapshotContents().Delete(ctx,
 					*volumeSnapshot.Status.BoundVolumeSnapshotContentName, metav1.DeleteOptions{})
 				gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
```

Comment:
can you create two wrapper methods,
1. delete snapshot content -> wait for it to be gone from CNS -> wait for pandora sync
2. delete snapshot -> wait for snapshot to be gone from CNS -> wait for pandora sync
this way next time you can just make minimal changes if required instead of doing it everywhere like we are doing now

Reply:
Done